magento/magento-functional-tests-migration
# 391 convert merge synonym group entity test mftf

### Description (*)
Convert MergeSynonymGroupEntityTest to MFTF

### Fixed Issues (if relevant)
1. #391 Convert MergeSynonymGroupEntityTest to MFTF

### Manual testing scenarios (*)
1. Open Backend.
2. Go to Marketing > Search Synonyms.
3. Click the "New Synonym Group" button.
4. Create new Synonym Group
5. Create duplicated Synonym Group and check merge
6. Create duplicated Synonym Group without merge enable option
7. Check the result 

### Contribution checklist (*)
 - [*] Pull request has a meaningful description of its purpose
 - [*] All commits are accompanied by meaningful commit messages
